targets/frontend/src/pages/user/password.js

```diff
@@ -20,7 +20,7 @@ export function ChangeMyPasswordPage() {
 
   return (
     <Layout title="Modifier mon mot de passe">
-      <PasswordForm onSubmit={handleChangePasword} />
+      <PasswordForm changeOldPassword onSubmit={handleChangePasword} />
```

Comment:
toujours true ça ?

Reply:
ok !

Reply:
les props valent `true` par default
https://fr.reactjs.org/docs/jsx-in-depth.html#props-default-to-true

Reply:
j'avais pas vu le nom du component ^^'